OpenWeb-SampleApp/Lifecycle/AppDelegate.swift

```diff
@@ -32,6 +33,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             .take(1)
             .subscribe()
 
+        let exampleStoreURL = "https://apps.apple.com/us/app/spotim-demo/id1234567"
```

Comment:
Let's move this block of code into a function at `AppCoordinator` file.
Call the function, `initialIAUSetup` from `initialSetup` function.

Reply:
done